docs/source/contributing/releasing.adoc

```diff
@@ -28,8 +21,38 @@ defined in your environment as _GITHUB_ACCESS_TOKEN_.
   For example, issues with a merged pull requests should be labeled as *resolution/done*.
 . Close milestone.
 
+[[auto-relase]]
+== Automated Release
+
+An automated release can be performed as:
+
+----
+$ export API_KEY=<api-key>
+$ export RELEASE_VERSION=<expected release version> # Eg : 1.0.0 (without 'v')
```

Comment:
export release version or milestone_id seems odd. 

BTW, I would expect the automated release to be integrated into Make, so that I do something like:
 
   make release RELEASE_VERSION=1.2.0 MILESTONE_ID=19 API_KEY=$API_KEY